### ISSUE TITLE
optimoroute integration

### ISSUE THREAD

**Issue by solomonbroadbent** [primal-kitchen/primal-kitchen#22]

**Comment by lestaku** [comment]
This would be cool

**Comment by solomonbroadbent** [comment]
yeah we can definitely do this. i had a bit of a look at the api.

anything marked as out-of-scope is just not part of the first build unless you're really needing it to be. so we can use that tag to make a list to future features etc 🙂
